charts/osm/values.yaml

```diff
@@ -4,7 +4,7 @@
 
 OpenServiceMesh:
   # -- `osm-controller` replicas
-  replicaCount: 1
+  replicaCount: 2
```

Comment:
If this is just for testing, could we hardcode `--set OpenServiceMesh.replicaCount=2` in the e2e framework vs. changing the default in values.yaml? Then regular installs will keep one by default.

Reply:
+1

Reply:
I think the argue last time was to ship the defaults we test (or at least, I remember saying that). I don't know if doing multi-replica would be a superset and catch all issues, including issues that would otherwise be more present in single replicaset. I don't have an answer.
@draychev ? ship HA by default or just change part/all testing to do multi-replica?

Reply:
Have you tested that all CLI commands and tooling work with multiple replicas? If not, let's set the default back to 1, to ensure we aren't breaking user facing features that currently do not handle multiple replicas of the controller.

Reply:
Good point. No I have not really tested any CLIs with multiple replica. I'll close this and maybe issue the change @nojnhuh suggested if the intention is to at least test multiple replica.